libnmstate/schema.py

```diff
@@ -244,6 +244,16 @@ class OVSBridge(object):
     STP = 'stp'
 
     PORT_SUBTREE = 'port'
-    PORT_NAME = 'name'
-    PORT_VLAN_MODE = 'vlan-mode'
-    PORT_ACCESS_TAG = 'access-tag'
+
+    class Port(object):
+        NAME = 'name'
+        VLAN_MODE = 'vlan-mode'
+        ACCESS_TAG = 'access-tag'
```

Comment:
This is not a related change and causes a collisions with other PR/s.
Please focus on the added new properties:
```
class Port:
    LINK_AGGREGATION_SUBTREE = 'link-aggregation'
    ...
```